subsys/fs/nvs/Kconfig

```diff
@@ -37,6 +37,14 @@ config NVS_DATA_CRC
 	  The CRC-32 is transparently stored at the end of the data field,
 	  in the NVS data section, so 4 more bytes are needed per NVS element.
 
+config NVS_BAD_PARTITION_RECOVERY
```

Comment:
What do you think about "NVS_INIT_BAD_REGION" ?

I would like this option and help text to reflect what it does and not (less) why it does this. NVS has no knowledge of partitions so using "partition" should be avoided.

The help could be something like:
Enable automatic initialization of a NVS on a flash region that does not contain a valid NVS. A region containing an invalid NVS can be caused by corruption or by providing a non-empty region. This option ensures a new NVS can be created.

Reply:
Makes sense, updated